### DISCUSSION TITLE
not able to customize units in parts anymore?

### DISCUSSION THREAD

**Discussion by anonkwe** [inventree/InvenTree#5054] [Q&A]
Hi, just came along this while trying to edit a part earlier: 
![image](https://github.com/inventree/InvenTree/assets/61838766/ed678716-fc67-4bb3-8b60-a15e9aafb87d)

before I had no problems using a custom unit, is there a new setting I'm missing?

thanks

**Comment by matmair** [comment]
I think this was limited in favour of auto-converting SI units; there is an FR for custom units: #4840

**Reply by SchrodingersGat** [reply]
Yep @anonkwe to support physical units (and especially for things like conversion of different supplier pack sizes) we have moved to a physical quantity based system.

**Reply by anonkwe** [reply]
is there a way to disable the validator maybe through inventree settings in the admin panel instead of having to use the backdoor after updates lol?

**Reply by anonkwe** [reply]
@SchrodingersGat is there a list of the valid units to use somewhere @SchrodingersGat

**Reply by SchrodingersGat** [reply]
Would make sense to have this in our documentation. For now, here's the standard list of supported units from the pint library:

https://github.com/hgrecco/pint/blob/master/pint/default_en.txt

**Comment by Flange0323** [comment]
Hi, has there been any further updates on this? was handy to be able to add custom units / not be limited

**Reply by wolflu05** [reply]
Yes, #5268 is merged.

**Comment by Flange0323** [comment]
Thanks for your reply

Does this mean there is now a way to add custom units in the latest
version? Or is is #5628 still ongoing?

Thanks, Matt

**Comment by SchrodingersGat** [comment]
@Flange0323 custom units will be available in the next minor release (0.13.0), or currently available in the latest version, available at https://demo.inventree.org
